CMakeLists.txt

```diff
@@ -1347,6 +1380,10 @@ if (USE_QT6)
       ${QT6QSCINTILLA_LIBRARY} ${Qt6DBus_LIBRARIES} ${Qt6Gamepad_LIBRARIES}
     )
   endif()
+  if(ENABLE_GUI_TESTS)
+    find_package(Qt6 REQUIRED COMPONENTS Test)
+    target_link_libraries(OpenSCAD PRIVATE Qt6::Test)
+  endif()
```

Comment:
I think it would be preferrable to merge this find_package invocation with the one that already looks for Qt6 (by adding `Test` to the list of components). Same for Qt5